htmd/parameterization/esp.py

```diff
@@ -3,15 +3,82 @@
 # Distributed under HTMD Software License Agreement
 # No redistribution in whole or part
 #
-
+import os
 from math import pi as PI
 from math import sqrt, sin, cos, acos
 import numpy as np
 from numpy.random import uniform as rand
+from scipy.spatial.distance import cdist
+import nlopt
+import unittest
+
 from htmd.molecule.vdw import radiusByElement
 
 class ESP:
-    # A temporary class to collect ESP stuff
+    """
+    Electrostatic potential (ESP) charge fitting
+
+    Capabilities
+    ------------
+    - Generate points
+    - Consider equivalent atoms
+    - Impose total molecule charge
+    - Impose charge values bounds
```

Comment:
I'd rather write: "Impose boundaries for charge values".

Reply:
Done!